pkgs/development/libraries/libfprint-thinkpad/default.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv
+, fetchFromGitHub
+, pkgconfig
+, meson
+, ninja
+, libusb
+, pixman
+, glib
+, nss
+, gtk3
+, coreutils
+, gtk-doc
+, docbook_xsl
+, docbook_xml_dtd_43
+, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libfprint-thinkpad";
+  version = "1.0";
+
+  src = {
+    libfprint-thinkpad =
+      fetchFromGitHub {
+        owner = "3v1n0";
+        repo = "libfprint";
+        rev = "2e2e3821717e9042e93a995bdbd3d00f2df0be9c";
+        sha256 = "1vps1wrp7hskf13f7jrv0dwry2fcid76x2w463wplngp63cj7b3b";
+      };
+  }.${pname};
+
+  nativeBuildInputs = [
+    pkgconfig
+    meson
+    ninja
+    gtk-doc
+    docbook_xsl
+    docbook_xml_dtd_43
+  ];
+
+  buildInputs = [
+    libusb
+    pixman
+    glib
+    nss
+    gtk3
+    openssl
+  ];
+
+  mesonFlags = [
+    "-Dudev_rules_dir=${placeholder "out"}/lib/udev/rules.d"
+    "-Dx11-examples=false"
+  ];
+
+  postPatch = ''
+    substituteInPlace libfprint/meson.build \
+      --replace /bin/echo ${coreutils}/bin/echo
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://fprint.freedesktop.org/;
+    description = "A library designed to make it easy to add support for consumer fingerprint readers";
+    license = licenses.lgpl21;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ abbradar ];
```

Comment:
I don't think abbradar should maintain this. Check the git history for who added this to the original expression.

Reply:
Based on the history and the file `maintainers/maintainer-list.nix` I've put `cizra` since he seems to be the one that has made the original thinkpad expression.
Not really sure on this one.

Reply:
I reviewed that original PR that added this and merged this. So I'm sure of this.
In hindsight, I'm not sure I should have merged it. Any user can perform these same overrides.
Though I don't think we can remove it, as cizra should be maintaining it at least.

Reply:
Okay, I keep it with Cirza as the maintainer.